django_mongodb/query_utils.py

```diff
@@ -8,7 +11,13 @@ def is_direct_value(node):
 def process_lhs(node, compiler, connection, bare_column_ref=False):
     if not hasattr(node, "lhs"):
         # node is a Func or Expression, possibly with multiple source expressions.
-        return [expr.as_mql(compiler, connection) for expr in node.get_source_expressions()]
+        result = []
+        for expr in node.get_source_expressions():
+            try:
+                result.append(expr.as_mql(compiler, connection))
+            except FullResultSet:
+                result.append(compiler.compile(Value(True)))
```

Comment:
well this thing is a little different from the others, because it returns some sql queries inside, and the other only returns the parameters. In this case the result may be: [("sql with an %s", [True]), ....other things]. Probably this is needed by some Django methods. So LGTM.

Reply:
I think what you're getting at is that this should be `Value().as_mql()` rather than `compile.compile()`.

Reply:
So so, I am just pointing that result will be a different type than the other result that process_rhs returns. And I want to know if this is needed. process_rhs normally returns parameters and this one also return the sql.

Reply:
In the fixed test, `node` is  `Coalesce(~Q(pk__in=[]), False)`. `result` is `[True, False]`.